code/game/mecha/mech_fabricator.dm

```diff
@@ -136,7 +136,7 @@
 	component_coeff = max(1 - (T - 1) / 4, 0.2) // 1 -> 0.2
 	for(var/obj/item/stock_parts/micro_laser/M in component_parts) //Not resetting T is intended; time_coeff is affected by both
 		T += M.rating
-	time_coeff = T / 2 // 1 -> 3
+	time_coeff = 2 / T // 1 -> 3
```

Comment:
<table>
<tr>
 <td>Before
 <td>Now
 <td>Suggested
<tr>
 <td>0.5
 <td>2
 <td>1.5
<tr>
 <td>1
 <td>1
 <td>1
<tr>
 <td>1.5
 <td>0.67
 <td>0.5
</table>
```suggestion
	time_coeff = 2 - ( T * 0.5) // T = 1 -> 3
```

Reply:
```suggestion
	time_coeff = 2 - ( T * 0.5) // 1 -> 3
```

Reply:
Had a discussion in discord about this

basically I explained how both the manipulator and laser influences construction time, and how at roundstart, T = 2, and tier 5s is T= 10 because both tiers are added to the total for T.

anyways we're sticking with 2 / T.